include/circt/Dialect/Calyx/CalyxControl.td

```diff
@@ -39,6 +39,10 @@ def ControlOp : CalyxContainer<"control", [
       region->push_back(new Block());
     }]>
   ];
+  let extraClassDeclaration = commonClassDeclaration # [{
```

Comment:
@mortbopet  Thanks for the advice,I have made the appropriate changes to what you said.To be honest, I don't really understand the style you gave inside the link, I may not understand what that means.But I think it might be used in a different scenes, you can see here.